apps/frontend/src/lib/electionDateUtils.ts

```
@@ -0,0 +1,132 @@
+    switch (format) {
+      case "short":
+        return `${month + 1}/${day}/${year}`;
+      case "long":
+        return electionDate.toLocaleDateString("en-US", {
+          timeZone: "UTC",
+          year: "numeric",
+          month: "long",
+          day: "numeric",
+        });
+      case "withOrdinal":
+        const ordinal = getOrdinal(day);
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)